lib/authorized_persona/authorization.rb

```diff
@@ -77,16 +77,16 @@ def authorized?
     private
 
     def authorization_current_user
-      unless authorization_current_user_method.is_a?(Symbol)
-        raise AuthorizedPersona::Error, "you must configure authorization with a valid current_user method name, " \
-                                        "e.g. `authorize_persona class_name: 'User', current_user_method: :my_custom_current_user`"
+      unless authorization_current_user_method.instance_of?(Symbol)
+        raise Error, "you must configure authorization with a valid current_user method name, " \
+                     "e.g. `authorize_persona class_name: 'User', current_user_method: :my_custom_current_user`"
```

Comment:
This branch can really only be reached if you set `klass.authorization_current_user_method =` _after_ having already called the `klass.authorize_persona` directive. I wasn't sure if this was really an intended feature, and if not then this `raise` is made redundant by the check on line 19.

However, I erred on the side of leaving lib code in place, rather than making a breaking change to remove (or make private) `#authorization_current_user_method=`.